src/AppInstallerCLICore/Argument.cpp

```diff
@@ -87,13 +87,17 @@ namespace AppInstaller::CLI
         case Args::Type::ExperimentalArg:
             return Argument{ "arg", NoAlias, Args::Type::ExperimentalArg, Resource::String::ExperimentalArgumentDescription, ArgumentType::Flag, ExperimentalFeature::Feature::ExperimentalArg };
         case Args::Type::Rename:
-            return Argument{ "rename", NoAlias, Args::Type::Rename, Resource::String::RenameArgumentDescription, ArgumentType::Positional, false };
+            return Argument{ "rename", 'r', Args::Type::Rename, Resource::String::RenameArgumentDescription, ArgumentType::Standard, false };
         case Args::Type::Purge:
             return Argument{ "purge", NoAlias, Args::Type::Purge, Resource::String::PurgeArgumentDescription, ArgumentType::Flag, false };
         case Args::Type::Preserve:
             return Argument{ "preserve", NoAlias, Args::Type::Preserve, Resource::String::PreserveArgumentDescription, ArgumentType::Flag, false };
         case Args::Type::Wait:
             return Argument{ "wait", NoAlias, Args::Type::Wait, Resource::String::WaitArgumentDescription, ArgumentType::Flag, false };
+        case Args::Type::PortableOverride:
+            return Argument{ "portable-force", NoAlias, Args::Type::PortableOverride, Resource::String::PortableOverrideArgumentDescription, ArgumentType::Flag, false };

```

Comment:
I would agree with making `--force` for general purpose and having specific security related overrides. This would allow for group policies that dictate which specific overrides users are allowed to perform. I could see a use case where a sysadmin may want to allow users to be able to use `--force` to allow overriding of ARP entries or of an output file, but wouldn't want to allow hash mismatches

Reply:
Removed portable-force argument. I use --force to handle this behavior and added a TODO statement to make sure we change it once --hash-override gets separated from --force.